docker-compose.queue.yaml

```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+services:
+  ffc-demo-payment-artemis-queue:
+    image: vromero/activemq-artemis:2.9.0-alpine
+    environment:
+      ARTEMIS_USERNAME: artemis
+      ARTEMIS_PASSWORD: artemis
+    ports:
+      - "8161:8161"
+      - "5672:5672"
```

Comment:
What value does the addition of override file bring?  It doesn't seem to do anything that isn't in `docker-compose.yaml` or `docker-compose.override.yaml`.

Reply:
Thanks, the file sneaked through I'll remove it.